cmd/gpud/command/command.go

```diff
@@ -252,11 +250,6 @@ sudo rm /etc/systemd/system/gpud.service
 					Usage: "enable 'kernel-module' component that returns healthy if and only if all the kernel modules are loaded (default: [], use '--kernel-modules-to-check=a --kernel-modules-to-check=b' for multiple modules)",
 					Value: &kernelModulesToCheck,
 				},
-				&cli.StringFlag{
-					Name:        "expected-port-states-nvidia-infiniband",
```

Comment:
we already use this flag on prod machines, right? we have to make sure the commandline flag must be removed in next release, otherwise it may fail to start because of unknown flag

Reply:
> we already use this flag on prod machines, right

Confirmed that, this is not being used. Safe to drop :)